multimu4e.el

```diff
@@ -180,6 +181,38 @@ If NAME or ADDRESS are not provided, use the variables `user-full-name' and
                     (or name user-full-name)
                     (or address user-mail-address)))))
 
+(defun multimu4e--change-signature-in-compose ()
+  "Change the signature of the current message.
+If the previous signature had no newline between the ending and
+the signature, this function removes the blank line before the
+new signature."
+  (save-excursion
+    (set (make-local-variable 'message-signature) mu4e-compose-signature)
```

Comment:
could you please add a #'boundp check before using `mu4e-compose-signature`? Your code should work beyond mu4e.